frontend/src/Screens/GlobalComponents/Navbar/Components/PageDesign/PageDesign.tsx

```diff
@@ -12,14 +12,7 @@ function PageDesign() {
                     <h2>Projects</h2>
                 </div>
```

Comment:
this is cant be compiled so i dont know how did you do that

Reply:
Please explain since the file compiles fine locally.